Mathlib/AlgebraicGeometry/EllipticCurve/Affine.lean

```diff
@@ -624,6 +624,35 @@ lemma nonsingular_add {x₁ x₂ y₁ y₂ : F} (h₁ : W.Nonsingular x₁ y₁)
   nonsingular_neg <| nonsingular_negAdd h₁ h₂ hxy
 #align weierstrass_curve.nonsingular_add WeierstrassCurve.Affine.nonsingular_add
 
+variable {x₁ x₂ : F} (y₁ y₂ : F) (hx : x₁ ≠ x₂)
+
+/-- The formula $x(P_1 + P_2) = x(P_1 - P_2) - \psi(P_1)\psi(P_2) / (x(P_2) - x(P_1)) ^ 2$,
+where $\psi(x,y) = 2y + a_1 x + a_3$. -/
+lemma addX_eq_subX_sub :
```

Comment:
I don't like this name, because `subX` isn't a thing. We either should make that a thing, or change this to `addX_sub_addX_negY_sub` (and I don't know what to call the rest).

Reply:
I renamed this lemma and the last one in https://github.com/leanprover-community/mathlib4/pull/13845/commits/2d333ea40f58745d5aa4c5c999f24a5ba72990b8 but leave the `cyclic_sum` one unchanged.

I think I answered all questions; thanks for the review!